Sources/Clappr/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -264,9 +268,17 @@ open class AVFoundationPlayback: Playback {
     }
 
     private func seekToStartAtIfNeeded() {
-        if canStartAt {
-            seek(startAt)
-        }
+        guard canStartAt else { return }
+        
+        seek(startAt)
+    }
+    
+    private func seekToLiveStartTime() {
+        guard let liveStartTime = liveStartTime else { return }
```

Comment:
There's no need to do this check here. You already check that in `canLiveStartTime`

Reply:
Swift does not allow make a calculation with a Double? and a Double, this guard statement is required here